internal/handlers/pg/msg_findandmodify.go

```diff
@@ -35,27 +32,14 @@ func (h *Handler) MsgFindAndModify(ctx context.Context, msg *wire.OpMsg) (*wire.
 		return nil, lazyerrors.Error(err)
 	}
 
-	// TODO https://github.com/FerretDB/FerretDB/issues/164
-
 	unimplementedFields := []string{
-		"sort",
-		"update",
 		"arrayFilters",
 		"let",
```

Comment:
could you please add here a `field` field to make sure it fails if present?

Reply:
Done.